src/index.html

```diff
@@ -11,6 +11,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     action="/"
+     method="post"
+     data-qa="big"
+     class="big-form"
+    >
+      <label
+        for="big-form-label"
```

Comment:
you don't need for when you wrapped input to label

Reply:
Ok, how should I do if I remove wrap
https://drive.google.com/file/d/1-5VeKwwvAFhArJLs9dpHv0jdTj9_hAjT/view?usp=share_link
https://drive.google.com/file/d/1yN_p5j1KSpr7LJzdhLTreskx0BeC5mN8/view?usp=share_link